docs/Style.md

```diff
@@ -91,3 +91,11 @@ var List = React.createClass({
 // ... in another file ...
 <List style={styles.list} elementStyle={styles.listElement} />
 ```
+## Supported Properties
+
+You can checkout latest support of CSS Properties in following Links.
+
+- [View Properties](https://github.com/facebook/react-native/blob/72d3d724a3a0c6bc46981efd0dad8f7f61121a47/Libraries/Components/View/ViewStylePropTypes.js)
+- [Image Properties](https://github.com/facebook/react-native/blob/72d3d724a3a0c6bc46981efd0dad8f7f61121a47/Libraries/Image/ImageStylePropTypes.js)
+- [Text Properties](https://github.com/facebook/react-native/blob/72d3d724a3a0c6bc46981efd0dad8f7f61121a47/Libraries/Text/TextStylePropTypes.js)
```

Comment:
Can you link to the documentation instead of the js files?

http://facebook.github.io/react-native/docs/text.html#style